daemons/fenced/fenced_remote.c

```diff
@@ -2284,58 +2259,58 @@ fenced_process_fencing_reply(xmlNode *msg)
         crm_notice("Action '%s' targeting %s using %s on behalf of %s@%s: %s%s%s%s",
                    op->action, op->target, device, op->client_name,
                    op->originator,
-                   pcmk_exec_status_str(result.execution_status),
-                  (result.exit_reason == NULL)? "" : " (",
-                  (result.exit_reason == NULL)? "" : result.exit_reason,
-                  (result.exit_reason == NULL)? "" : ")");
+                   pcmk_exec_status_str(op->result.execution_status),
+                  (op->result.exit_reason == NULL)? "" : " (",
```

Comment:
one more space of indentation on these three lines (was off before this PR, so your call)

Reply:
Hopefully there will be a follow-up not too far off, to build this as a GString, so I'll let it be until then